gem/settings/base.py

```diff
@@ -63,15 +63,6 @@
 if USE_OIDC_AUTHENTICATION:
```

Comment:
what about passing the theme to the login url or should we do that in another PR??

Reply:
Did that in a separate PR, it is handled differently to everything else